srcalc/src/test/java/gov/va/med/srcalc/domain/model/MultiSelectOptionTest.java

```diff
@@ -25,4 +26,23 @@ public final void testEquals()
             .verify();
     }
     
+    @Test(expected = IllegalArgumentException.class)
+    public final void testValueEmpty()
+    {
+        new MultiSelectOption("");
+    }
+    
+    @Test(expected = IllegalArgumentException.class)
+    public final void testValueTooLong()
+    {
+        new MultiSelectOption(
+                TestHelpers.stringOfLength(MultiSelectOption.VALUE_MAX + 1));
+    }
+    
+    @Test(expected = IllegalArgumentException.class)
+    public final void testValueInvalidCharacters()
+    {
+        new MultiSelectOption("foo\t");
+    }
```

Comment:
Meh, the default constructor is not really part of the public API so I don't want to test it. It's only for Hibernate's reflection-based construction.